### PR TITLE
Fix components in new Webpack setup, add Dev server

### PR DESCRIPTION
**Pull request by Jakubem**
also switched back to regular `ts-loader`

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+9.8.0
```
Comment (Worie) [inline .nvmrc]: Can we use 9.2.0? Can you check if it'll run on it?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```
Comment (Worie) [inline app.js]: what is this? can you explain is it necessary?

File: package.json [hunk 3]
```diff
@@ -1,96 +1,41 @@
 {
-  "name": "koduje-spolecznosc",
-  "description": "A Vue.js project",
+  "name": "kie-webpack-setup",
```
Comment (Worie) [inline package.json]: lets go back to koduje-spolecznosc please (and overall, package.json should seem legit)

File: src/components/atoms/a-button/a-button.ts [hunk 4]
```diff
@@ -1,9 +1,9 @@
 import { Component, Vue } from 'vue-property-decorator'
 import './a-button.scss'
+import template from './a-button.html'
```
Comment (Worie) [inline src/components/atoms/a-button/a-button.ts:3]: thanks

**Review by Worie — CHANGES_REQUESTED** [review]
please have a look, there are only minor issues / requests

File: .nvmrc (hunk 1 above)
Comment (Jakubem) [reply .nvmrc]: On it!

File: app.js (hunk 2 above)
Comment (Jakubem) [reply app.js]: Oh dang! Sorry for that. I totally forgot about it.

File: package.json (hunk 3 above)
Comment (Jakubem) [reply package.json]: On it!

File: src/components/atoms/a-button/a-button.ts (hunk 4 above)
Comment (Jakubem) [reply src/components/atoms/a-button/a-button.ts:3]: np

File: .nvmrc (hunk 1 above)
Comment (Jakubem) [reply .nvmrc]: It seems to be working

**Comment by Worie** [comment]
thank you then, good job!

File: src/components/atoms/a-button/a-button.ts (hunk 4 above)
Comment (Jakubem) [reply src/components/atoms/a-button/a-button.ts:3]: it's all fixed now

**Pull request merged**